--- app/node_modules/react-native/third-party-podspecs/glog.podspec	2024-03-25 11:16:21
+++ app/node_modules/react-native/third-party-podspecs/glog.podspec 2024-03-25 11:16:53
@@ -45,7 +45,7 @@
   spec.pod_target_xcconfig = {
     "USE_HEADERMAP" => "NO",
     "HEADER_SEARCH_PATHS" => "$(PODS_TARGET_SRCROOT)/src",
-    "DEFINES_MODULE" => should_define_modules # When the workaround is removed, set this var to "YES"
+    "DEFINES_MODULE" => "YES" # When the workaround is removed, set this var to "YES"
   }

   # Pinning to the same version as React.podspec.
